app/digests/tests/test_digest_api.py

```diff
@@ -38,13 +43,20 @@ def test_has_expected_data_in_response(client: Client,
 def test_can_get_digest_by_id(client: Client,
                               digest: Digest,
                               digest_json: Dict):
-    response = client.get(f'/digests/{digest.id}')
+    response = client.get(f'{DIGESTS_URL}/{digest.id}')
     assert response.data == digest_json
-    assert response.content_type == settings.DIGESTS_CONTENT_TYPE
+    assert response.content_type == settings.DIGEST_CONTENT_TYPE
 
 
 @pytest.mark.django_db
 def test_has_digest_content_type_header(client: Client):
-    response = client.get('/digests')
+    response = client.get(DIGESTS_URL)
     assert response.status_code == 200
     assert response.content_type == settings.DIGESTS_CONTENT_TYPE
+
+
+@pytest.mark.django_db
+def test_can_ingest_digest(rest_client: APIClient, digest_json: Dict):
+    response = rest_client.post(DIGESTS_URL, data=json.dumps(digest_json),
+                                content_type=settings.DIGEST_CONTENT_TYPE)
```

Comment:
https://github.com/elifesciences/digests/blob/f44a30b832b45a4185b1ed152c741d1b730de16c/app/digests/tests/conftest.py#L164-L179 where `digest_json` comes from correctly doesn't have `stage`. It has a `published` which in this use case shouldn't be used because it would appear only later in the `PATCH`. Should probably unset it here before the request, or specify directly the full body of the request for clarity

Reply:
Will add in `stage` to the fixture.

I left `published` in due to @gnott 's comment:

> The published date will get set if not already available prior to the POST.

I read this as it is possible (though now not required) that the published date could appear in the POST data.

Perhaps needs clarifying.